src/App.js

```diff
@@ -1,11 +1,32 @@
 import React from 'react';
 import './App.css';
+import { Route, Switch, NavLink } from 'react-router-dom';
+import Tabs from './components/Tabs/Tabs';
 
 const App = () => (
-  <div className="App">
-    <HomePage />
-    <TabsPage />
-  </div>
+  <>
+    <nav className="ui pointing menu">
+      <NavLink
+        exact
+        to="/"
+        activeClassName="active item"
```

Comment:
`activeClassName="active"` - will be enough

Reply:
it's name class from semantic UI